app/models/qdm/basetypes/data_element.rb

```diff
@@ -1,3 +1,4 @@
+require_relative('../id.rb')
```

Comment:
In models.rb the qdm/id needs to be moved before require_relative 'qdm/basetypes/data_element'

for example, (app/models/models.rb) could be modified to look something like this

require 'mongoid'

# Generated models with base type dependencies
require_relative 'qdm/id'

# base types
require_relative 'qdm/basetypes/code'
require_relative 'qdm/basetypes/interval'
require_relative 'qdm/basetypes/quantity'
require_relative 'qdm/basetypes/data_element'